packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_inspector_platform.dart

```diff
@@ -132,6 +128,20 @@ abstract class GoogleMapsInspectorPlatform extends PlatformInterface {
     throw UnimplementedError('getHeatmapInfo() has not been implemented.');
   }
 
+  /// If the platform supports getting information about ground overlays.
+  bool supportsGettingGroundOverlayInfo() => false;
```

Comment:
This follows same pattern as heatmap implementation.
Having this method could be unnecessary, as all platforms support getting groung operlay info at the moment.
We can remove this method seen valid change.

Reply:
I like that we're adding these capability checks to plugins, we should do this more.

Reply:
Yep, doing this more is actually our written policy now :)